jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -254,6 +287,35 @@ def _jwst2data(file_obj, ext, data_label):
 
     return data, new_data_label
 
+# ---- Functions that handle input from Roman ASDF files -----
+
+
+def _roman_2d_asdf_to_glue_data(file_obj, data_label, ext=None):
+
+    if ext == '*':
+        ext_list = ['data', 'dq', 'err', 'var_poisson', 'var_rnoise']
+    elif isinstance(ext, list):
+        ext_list = ext
+    else:
+        ext_list = [ext]
+
+    for ext in ext_list:
+        comp_label = ext.lower()
+        new_data_label = f'{data_label}[{comp_label}]'
```

Comment:
Should this use Jesse's label generator?

Reply:
Imviz has its own rules since it can load wildcard extension and have both FITS and ASDF support, and so on. If the function is hidden, doesn't hurt if you also want to pass in `app` to them, but becareful to not break current Imviz labeling behavior. 🤪